org.kde.dragonplayer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "org.kde.dragonplayer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.9",
```

Comment:
kde is in process of updating all apps to 6.10 so you may do it as well. don't forget removing ffmpeg-full if you do this